### DISCUSSION TITLE
Formula moved from the tap into homebrew/core

### DISCUSSION THREAD

**Discussion by amalota** [Homebrew/discussions#3712] [Tap maintenance and Homebrew development] [answered]
Hi,

So we just recently got our tool included in the `homobrew-core` repository and we were wondering whether users which installed our tool from the `tap` will get updates pulled from the `tap` or from the `homebrew/core`, because if the updates will get pulled from the `tap` it would mean that we would still have to maintain it for the older users.

**Comment by carlocab** [comment, marked as answer]
You need to add the relevant formula to your tap's `tap_migrations.json`. See https://docs.brew.sh/Migrating-A-Formula-To-A-Tap and https://github.com/universal-ctags/homebrew-universal-ctags/blob/c84df0fc89f3a63ce5438eb59244ec2080352c0d/tap_migrations.json for an example.

**Reply by amalota** [reply]
@carlocab Works like a charm, thanks!
